components/automate-cli/pkg/verifyserver/services/certificatevalidation/certificatevalidation.go

```diff
@@ -207,14 +207,17 @@ func (vc *ValidateCertificateService) CertificateValidation(req models.Certifica
 	var response = models.CertificateCheckResponse{}
 
 	// certKeys and keys arrays are required to maintain the order of correct response messages.
-	certKeys := []string{constants.ROOT, constants.NODE}
+	certKeys := []string{constants.NODE}
 	keys := []string{constants.NODE_KEY}
 
 	// certificates map is used for storing Root Certificate, Node Certificate and if it is a case of opensearch then Admin Certificate.
 	certificates := make(map[string]string)
 	// privateKeys map is used for storing Private Key and if it is a case of opensearch then Admin Private Key.
 	privateKeys := make(map[string]string)
-	certificates[constants.ROOT] = req.RootCertificate
+	if strings.TrimSpace(req.RootCertificate) != "" {
```

Comment:
This will change the sequence of keys array. We have to take care of root cert first then node cert

Reply:
@Sahiba3108 , added in correct order. :)